packit_service/worker/helpers/build/build_helper.py

```diff
@@ -652,6 +652,9 @@ def report_status_to_all_test_jobs(
         for test_job in self.job_tests_all:
             if test_job.skip_build:
                 continue
+            elif test_job.manual_trigger:
+                if state not in (BaseCommitStatus.error, BaseCommitStatus.failure):
```

Comment:
I thought that showing errors can make it easier for a user to understand why tests are not running, but then I realized that when the copr build will work the failed manual test check will not be overwritten. And this is misleading. So it is probably better never to show these check statuses at all. WDYT?

Reply:
I would rather be consistent and really hide all the statuses